### ISSUE TITLE
avcodec_receive_frame could not be located in nw.dll

### ISSUE THREAD

**Issue by richard-scott** [betaflight/blackbox-log-viewer#101]
I've not used the log viewer in a few days, but today's build direct from master gives me this error.

![2018-01-25 20_39_16-betaflight-blackbox-explorer exe - entry point not found](https://user-images.githubusercontent.com/5149834/35411347-5f0abdc8-0210-11e8-8c5c-f83cba39ae8d.png)

I'm using the same build process as before which is still working for the configurator.

```
cd blackbox-log-viewer
call git reset --hard
call git pull
call npm install
call gulp apps --win32
```

**Comment by McGiverGim** [comment]
What windows version do you have?

**Comment by richard-scott** [comment]
Windows 10, 64bit.
Last time I built this looks to have been on the 18th and it all worked then.
I can't build a 64bit version as there is no option for that.

**Comment by McGiverGim** [comment]
Yes, you can build 64 bits version. It was merged some days ago. Please test it.

**Comment by richard-scott** [comment]
ah it seems that a "git reset --hard" doesn't clean out files created in the last build... I need to use "git clean --force" before my "git pull". All good now :-)

**Comment by McGiverGim** [comment]
Is better that you let the issue open. The official version is the 32 bits one. The 64 bits is not officially supported.

**Comment by richard-scott** [comment]
The issue is gone now tho... If it comes back I'll open a new case. This one was user error 😀

**Comment by richard-scott** [comment]
Ah, this one issue is back but this time I tried with: gulp apps --win32
I'm attempting to setup a new PC to build on.

**Comment by McGiverGim** [comment]
Yes, working in a patch. Same issue than here: https://github.com/betaflight/blackbox-log-viewer/issues/130

**Comment by McGiverGim** [comment]
Fixed here: https://github.com/betaflight/blackbox-log-viewer/pull/131
